source/amge_host.cc

```diff
@@ -9,7 +9,26 @@
  * SPDX-License-Identifier: BSD-3-Clause                                 *
  *************************************************************************/
 
+#include <mfmg/adapters_dealii.hpp>
 #include <mfmg/amge_host.templates.hpp>
 #include <mfmg/instantiation.hpp>
 
-INSTANTIATE_DIM_VECTORTYPE(TUPLE(AMGe_host))
+// INSTANTIATE_DIM_VECTORTYPE(TUPLE(AMGe_host))
```

Comment:
You need to embrace the macro, be one with the macro, and only then the macro will accept you.